Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+ARG NODE_VERSION
```

Comment:
Don't. Parametrizing `Dockerfile` base images is a bad idea. It tends to get out of hand really quickly and you just don't really know what the image is based upon in a particular point in time.

Reply:
Ah, I forgot, I think we wanted to add base node version to tags. So basically something like
`:1.2.3-node-17, :1.2.3, :latest` Would that be ok?

Reply:
Okay, but if you use different base images - have different dockerfiles. There will be differences in the necessary dependencies of you target different OSes (or different version of OS, and base image will eventually do that).